hail/src/main/c/Upcalls.cpp

```diff
@@ -35,6 +35,19 @@ UpcallConfig::UpcallConfig() {
   auto cl2 = env->FindClass("is/hail/io/InputBuffer");
   InputBuffer_close_ = env->GetMethodID(cl2, "close", "()V");
   InputBuffer_readToEndOfBlock_ = env->GetMethodID(cl2, "readToEndOfBlock", "(J[BII)I");
+  // OutputStream methods
+  auto cl3 = env->FindClass("java/io/OutputStream");
+  OutputStream_write_ = env->GetMethodID(cl3, "write", "([BII)V");
+  OutputStream_flush_ = env->GetMethodID(cl3, "flush", "()V");
+  // OutputStream methods
+  auto cl4 = env->FindClass("is/hail/io/OutputBlockBuffer");
```

Comment:
yeah, that's the goal. I hadn't done that in this pass because the OutputBlockBuffer -> OutputStream level needs the lz4 stuff, but I'll figure that out before I actually PR the encoder.